Simple solvers/EG.py

```diff
@@ -43,10 +43,19 @@
     
     
 else:
-    xint=(-b+(dis)**0.5) / (2*a)
-    print("There is one possible solution")
+    xint=(-b+(discriminant)**0.5) / (2*a)
+    print("There is one possible solution when discriminant is equal to zero (b**2-4ac)")
     print(xint)
-    
+    print ("verifying using different method")
```

Comment:
Some comments might help the ready figure out what's going on... take a look at `JV.py` as an example.

Reply:
I'll add some comments to it>